cmd/license-lint/collector.go

```diff
@@ -127,8 +127,13 @@ func getDependentModules() ([]module, error) {
 	cmd.Stderr = &stderr
 
 	err := cmd.Run()
+	errstr := stderr.String()
+	if strings.Contains(errstr, "not part of a module") {
```

Comment:
I am a little concerned about using hardcoded text from returned errors, but it looks like Golang tests are also using the hardcoded string so hopefully it is less likely to change.